app/helpers/settings_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SettingsHelper
```

Comment:
Consider using [this concern](https://github.com/internetee/auction_center/blob/master/app/models/concerns/format_validator.rb) , it's in `master` now.